src/Terrain/MapMaster.java

```diff
@@ -33,8 +33,7 @@ public MapMaster(CommandingOfficers.Commander[] COs, MapInfo mapInfo)
     }
 
     // Print a warning if the number of Commanders we have does not match the number the map expects.
-    if( COs.length != mapInfo.COProperties.length ||
-      ( mapInfo.mapUnits.size() > 0 && COs.length != mapInfo.mapUnits.size() ) )
```

Comment:
If you're removing this, you might also wanna set initOK to false in the other if statement?